Non-custodial-Lightning-Network-Solutions.md

```diff
@@ -0,0 +1,36 @@
+| Author | Phoenix Green |
+| :---- | :---- |
+| Title | Non-custodial Lightning Network Solutions (Better Title needed) |
+| Headline  | featured text |
+| Cypherpunktimes | cypherpunktimes.com/...link |
+| Publish Date | 00 Jan 20?? |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | post link… |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement 
+
+The Lightning Network has achieved the monumental target of processing thousands of micro-transactions per second at the cost of a single Atom/Satoshi. But is it possible to have this functional in a non-custodial wallet?
+
+Pre-discussion, research and questions:
```

Comment:
I like the topic. Maybe it would also be good an intro about the revolutionary LN technology and which crypto networks already implemented it

Reply:
Good points. Other than Decred, Litecoin and Bitcoin BTC I'm not aware of any other platforms that are developing / integrating the Lightning Network?